### DISCUSSION TITLE
Proposal to Include "ComfyUI-LexMSDBNodes" Custom Node in the Manager Database

### DISCUSSION THREAD

**Discussion by SOELexicon** [Comfy-Org/ComfyUI-Manager#28] [Show and tell]
Hello everyone,

I have been working on a custom node specifically designed to work with MSSQL databases. This node, which is part of the "ComfyUI-LexMSDBNodes" repository, is something that I believe could be beneficial to the wider community. Therefore, I propose its addition to the manager database.

Repository Information:
Repository Name: ComfyUI-LexMSDBNodes
GitHub Link: [ComfyUI-LexMSDBNodes GitHub URL](https://github.com/SOELexicon/ComfyUI-LexMSDBNodes)
Brief Description: This repository contains the necessary nodes to interact with MSSQL databases from within ComfyUI. It provides intuitive and comfortable functionality, bridging the gap between ComfyUI and MSSQL databases.

### Custom Node Details:
**Node Name**: ComfyUI-LexMSDBNodes

**Functionality**: These nodes facilitate seamless interaction between ComfyUI and MSSQL databases. They provide various functionalities, such as reading config, establishing connections, executing SQL queries, and manipulating data.

**How it can benefit the community:** The addition of these nodes would simplify image storage management, info storage for ComfyUI . They can leverage these nodes to execute complex SQL operations without needing deep technical knowledge.
I believe that the inclusion of the "ComfyUI-LexMSDBNodes" custom node into the Manager DB would greatly enhance productivity and offer new, practical functionality to our projects.

other projects could potentially hook into the classes of this node package to also store information/images etc if they wished to implement such a feature

I'm open to answering any questions you may have about my proposal or the custom node. I'm also happy to make any necessary modifications to better suit the community's needs. its still a work in progress the main 2 nodes are 

Thank you for considering my proposal. I'm eagerly looking forward to hearing your feedback!

**Comment by ltdrdata** [comment]
I'd added this.
